lua/nvim-tree/renderer/init.lua

```diff
@@ -51,6 +51,8 @@ local picture_map = {
   jpeg = true,
   png = true,
   gif = true,
+  webp = true,
+  jxl = true,
```

Comment:
Hrmmm....

I was planning to remove this very-legacy mapping as it should be handled by nvim-web-devicons.

https://github.com/nvim-tree/nvim-web-devicons/pull/318 has been merged. Is that sufficient @zootedb0t ?

Reply:
Icon for `jxl` files work now, but the highlight still doesn't apply to `jxl and webp` files.

![Screenshot_2023-10-08-05-59-25_1920x1080](https://github.com/nvim-tree/nvim-tree.lua/assets/62596687/01e08f30-c20d-4b63-80e4-06fa224e27c5)

Reply:
Ah I see. I forgot about highlight...